stable/vsphere-cpi/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.0.0
```

Comment:
Does this need to be updated to v1.1.0 as well?

Reply:
Good catch. Updated.